test/ModuleInterface/back-deploy-attr.swift

```diff
@@ -0,0 +1,40 @@
+// RUN: %empty-directory(%t)
+
+// Ensure @_backDeploy attributes and function bodies are printed in
+// swiftinterface files.
+// RUN: %target-swift-frontend -emit-module -o %t/Test.swiftmodule -emit-module-interface-path %t/Test.swiftinterface -module-name Test %s
+// RUN: %FileCheck %s --check-prefix FROMSOURCE --check-prefix CHECK < %t/Test.swiftinterface
```

Comment:
I like to verify the swiftinterfaces that the compiler emits whenever possible. Just for this it might be worth invoking the driver instead of the frontend and passing `-verify-emitted-module-interface`.

Reply:
Good idea, fixed